messaging/kafka-streams-reactive-messaging/src/test/resources/application.properties

```diff
@@ -0,0 +1,43 @@
+# Configuration file
+quarkus.test.native-image-profile=test
+quarkus.http.port=8080
+quarkus.http.test-port=8080
+
+producer.httpCodes=200,201,400,401,403,500
+producer.loginUrls=redhat/login,inditex/login,santander/login,bbva/login
+sse.endpoint=http://localhost:8080/monitor/stream
+
+kafka.bootstrap.servers=kafka-broker-1:9092
+%test.kafka.bootstrap.servers=localhost:9092
```

Comment:
As far I see, we don't need the test profile. When running locally, the test resource will overwrite this property setting the correct settings.